flexwrfoutput/postprocess.py

```diff
@@ -0,0 +1,102 @@
+"""
+Additional functions needed in preprocess to secure compatibility to xWRF
+"""
+from datetime import datetime
+
+import numpy as np
+import xarray as xr
+
+
+def _prepare_conc_units(ds: xr.Dataset) -> xr.Dataset:
+    """
+    Change units of footprint data to be pint compatible.
+    """
+    ds.CONC.attrs["units"] = (
+        "s m^3 kg^-1" if ds.CONC.units == "s m3 kg-1" else ds.CONC.units
+    )
+    return ds
+
+
+def _make_attrs_consistent(ds: xr.Dataset) -> xr.Dataset:
+    """
+    Change attribute names and values of FLEXPART-WRF output to be compatible with xWRF
+        postprocess.
+    """
+
+    ndims_lat = ds.dims["south_north"]
+    ndims_lon = ds.dims["west_east"]
+
+    lat_center_index = (
+        int(ndims_lat // 2)
+        if ndims_lat % 2 == 1
+        else [int(ndims_lat // 2), int(ndims_lat // 2 - 1)]
+    )
+    lon_center_index = (
+        int(ndims_lon // 2)
+        if ndims_lon % 2 == 1
+        else [int(ndims_lon // 2), int(ndims_lon // 2 - 1)]
+    )
+
+    ds.attrs["CEN_LAT"] = (
+        ds.XLAT.isel(south_north=lat_center_index, west_east=lon_center_index)
+        .mean()
+        .values
+    )
+    ds.attrs["CEN_LON"] = (
+        ds.XLONG.isel(south_north=lat_center_index, west_east=lon_center_index)
+        .mean()
+        .values
+    )
```

Comment:
```suggestion
    assert (ds.south_north == list(range(ds.dims['south_north']))).all()
    assert (ds.west_east == list(range(ds.dims['west_east']))).all()

    y_center_index = (ds.dims["south_north"] - 1) / 2
    x_center_index = (ds.dims["west_east"] - 1) / 2
   
    ds.attrs["CEN_LAT"] = (
        ds.XLAT.interp(south_north=y_center_index, west_east=x_center_index).item()
    )
    ds.attrs["CEN_LON"] = (
        ds.XLONG.interp(south_north=y_center_index, west_east=x_center_index).item()
    )
```

Ist ein bisschen cuter so ;)

Reply:
Done in dc9935ef69d334fb875046980d55efa7145bbb8b